Client/core/CCore.h

```diff
@@ -40,6 +40,7 @@ class CCore;
 #include <ijsify.h>
 #include <core/CWebCoreInterface.h>
 #include "CTrayIcon.h"
+#include "CDiscordManager.h"

```

Comment:
To improve compile speed, please use a forward declaration for this and include `CDiscordManager.h` in `CCore.cpp`